lang/cs/Org.Apache.REEF.Driver/Constants.cs

```diff
@@ -32,12 +31,6 @@ public static class Constants
         /// The class hierarchy file from .NET.
         /// </summary>
         [Obsolete("Deprecated in 0.14. Will be removed in 0.15.")]
-        public const string ClassHierarachyBin = "clrClassHierarchy.bin";
-
-        /// <summary>
-        /// The class hierarchy file from .NET.
-        /// </summary>
-        [Obsolete("Deprecated in 0.14. Will be removed in 0.15.")]
         public const string ClassHierarchyBin = "clrClassHierarchy.bin";
```

Comment:
To be frankly with you, I tried to do those together during making this PR, but I found that it's not so easy. I think it's beyond the scope of original intention of REEF-842.
It might be marked deprecated by you, @tcNickolas, in other commits.
I think you know how to handle those properly and do that in another PR.
We cannot remove all deprecation from other committers in one PR. There are many kinds of deprecations. If you allow me, I want to close this PR at the original scope. :)